cdc/processor/sinkmanager/table_sink_worker.go

```diff
@@ -155,7 +155,7 @@ func (w *sinkWorker) handleTask(ctx context.Context, task *sinkTask) (finalErr e
 				// All transactions before currTxnCommitTs are resolved.
 				err = w.advanceTableSink(task, currTxnCommitTs, committedTxnSize+pendingTxnSize)
 			} else {
-				// This means all events of the currenet transaction have been fetched, but we can't
```

Comment:
Actually, I try to fix this typo, `currenet` to `current`, but remove it by mistake.

But thi should still be ok, because `the` is already indicate `current` in this sentence.

Reply:
it's fixed.